### ISSUE TITLE
Current documentation is not correct

### ISSUE THREAD

**Issue by bwscheller** [MicrosoftDocs/vsts-docs#581]
Prior to the TFS 2018 Update 1, I was able to delete Git repositories.  Now, when selecting "Manage Repositories" from either repositories pulldown within Code or under the Code menu, I am taken to Security for all Git repositories.  This screen does not contain a list of repositories and there is no kebab that allows the selection of the Delete repository option.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b03af1d9-4eeb-ac18-d517-a0a44a2478be
* Version Independent ID: 842691c8-8c41-facf-2cff-03542262f2e5
* Content: [Delete a Git repo from your team project](https://docs.microsoft.com/en-us/vsts/git/delete-existing-repo?view=vsts)
* Content Source: [docs/git/delete-existing-repo.md](https://github.com/MicrosoftDocs/vsts-docs/blob/live/docs/git/delete-existing-repo.md)
* Service: **unspecified**
* GitHub Login: @steved0x
* Microsoft Alias: **sdanie**

**Comment by Rick-Anderson** [comment]
Delete from projects page. Not very discoverable
![image](https://user-images.githubusercontent.com/3605364/39342630-917d9d24-4974-11e8-9f15-31db97098819.png)

**Comment by bwscheller** [comment]
You have provided directions for deleting a TFS Project, I would like to delete a Git Repository within a TFS Project.

**Comment by bwscheller** [comment]
I just found it.  The left most pane was collapsed.  Expanding it displayed a list of all Git repositories with a kebab that allows them to be deleted.  May want to add a note to ensure that the left most repositories pane is expanded in the documentation here: https://docs.microsoft.com/en-us/vsts/git/delete-existing-repo?view=vsts

**Comment by wolivier** [comment]
Thanks, Rick-Anderson, so indeed under Config icon -> Overview.  Not available from the main projects list at first glance, so got to agree, irritatingly hidden.

**Comment by steved0x** [comment]
@bwscheller We will get this updated, thanks for letting us know and I'm glad you got it worked out. Thank you for the feedback!

**Comment by Rick-Anderson** [comment]
@steved0x  please open this until it's fixed. WHen it's open other folks can see the issue and the work around.

**Comment by Rick-Anderson** [comment]
It's tricky to get to the right projects page.

* Go to the root of the projects `https://{URL like abc1111122}.visualstudio.com/_projects`
* Hover over the gear icon > **Overview**

**Comment by steved0x** [comment]
@bwscheller Glad you got it worked out and I have added a note to ensure the repositories pane is expanded. @wolivier @Rick-Anderson I have updated the screenshots for clarity and with the extra note about expanding the repositories pane, and reviewed the other steps, which are correct, but hopefully combined with the improved screenshots, hopefully the process is more clear. Thank you for the feedback. These changes should go live later today.
